awswrangler/oracle.py

```diff
@@ -457,4 +458,13 @@ def handle_oracle_objects(
                 Decimal(repr(col_value)) if isinstance(col_value, float) else col_value for col_value in col_values
             ]
 
+        # A user may wish to represent Oracle blob binary data as plain text - if so base64 encode bytes
```

Comment:
This sounds like a feature - is that a fairly common use case? while the overall PR looks good (thanks a lot!) I'm not sure about this specific behaviour...

Reply:
> This sounds like a feature - is that a fairly common use case? while the overall PR looks good (thanks a lot!) I'm not sure about this specific behaviour...

Hi @cnfait ,

Thanks for taking a look! Yes that is true, in essence representing binary as a string is a form of transformation which can be viewed as a new feature/behaviour. Happy to drop it if that sits better with you, simply remove that block. 

Also my apologies for not being able to get the final check passing. I've not been able to stand up a full dev environment to run through the tests.

Reply:
Hi @cnfait , I've pushed a commit which reverts this behaviour.

Reply:
Thanks again!